main.go

```diff
@@ -108,7 +113,7 @@ func configure() {
 	domains.Configure()
 	jwtmanager.Configure()
 	cookie.Configure()
-	responses.Configure()
+	responses.Configure(templatesFs)
```

Comment:
see the note regarding `Configure()` in `main.go`.  I'm hopeful of one day all `Configure()` calls becoming `init()`.  As such, we shouldn't pass an argument to `Configure()` calls.

Reply:
will look into this

Reply:
one of the issues is that I can't go:embed files from a parent directory, so that's why `//go:embed templates` is in main.go. if it needs to go in responses.go, the template files would need to be moved there.

an alternative approach would be to have both `init()` AND `configure(templateFS)`

What do you think?

Reply:
here's one approach 8ca24f1ecde44f18416ee874a8966e5273c71b6b